GenomeFileUtil.spec

```diff
@@ -62,8 +65,48 @@ module GenomeFileUtil {
         string genome_ref;
     } GenomeSaveResult;
 
+    typedef structure {
+        string genome_ref;
+        string assembly_ref;
+        string assembly_path;
+        Workspace.object_info assembly_info;
+        Workspace.object_info genome_info;
+    } GenbankToGenomeSaveResult;
+
     funcdef genbank_to_genome(GenbankToGenomeParams params)
-                returns (GenomeSaveResult result) authentication required;
+                returns (GenbankToGenomeSaveResult result) authentication required;
```

Comment:
This change is backwards incompatible and may break other code, as the structure names get compiled into Java classes.

Reply:
My concern is my modification on GenomeSaveResult (now return object_info on genome && assembly) would impact other function that also use GenomeSaveResult as its return.

Reply:
Only use GenbankToGenomeSaveResult for the bulk function, in that case, and leave the single function alone

Reply:
ah, the single function will use _import_genbank_mass after setup, which returns a bunch of new stuff.

Reply:
Take it out of the return then

Reply:
👍

Reply:
:+1: